compiler/qsc_ast/src/assigner.rs

```diff
@@ -9,7 +9,7 @@ use crate::{
     mut_visit::{self, MutVisitor},
 };
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Does this still need to be clone-able? I don't see anywhere else in the changes that adds cloning for it, but I might be missing something. Cloning the assigner can be risky; it's safe as long as all AST handled by the cloned assigner is thrown away, but if it co-exists with AST handled by the original assigner it can open up the possibility of node id collisions.

Reply:
Hmm, this seems to be an accident, thanks for the catch!